src/opers.c

```diff
@@ -678,7 +678,7 @@ Obj FuncAND_FLAGS (
 
     /* check the cache                                                     */
 #   ifdef AND_FLAGS_HASH_SIZE
-        if ( INT_INTOBJ(flags1) < INT_INTOBJ(flags2) ) {
+        if ( flags1 < flags2 ) {
```

Comment:
I'm going to go back and look at this again. `INT_INTOBJ` never made sense, as `flags` isn't an `INTOBJ`, it's just a normal bag. That means it was just comparing memory locations, so I haven't changed the behaviour of the code.